### ISSUE TITLE
d2l-table causes aria-hidden-focus accessibility violation

### ISSUE THREAD

**Issue by Vitalii-Misechko** [BrightspaceUI/core#1362]
```
Rule: aria-hidden-focus
      Impact: serious
      ARIA hidden element must not contain focusable elements (https://dequeuniversity.com/rules/axe/4.2/aria-hidden-focus?application=axeAPI)
      
      Issue target: d2l-insights-assessment-quality-dashboard,d2l-table-wrapper,d2l-scroll-wrapper,.d2l-scroll-wrapper-button-left
      Context: <button class="d2l-scroll-wrapper-button d2l-scroll-wrapper-button-left" aria-hidden="true">     
                                        <d2l-icon icon="tier1:chevron-left"></d2l-icon>
                                </button>
      Fix all of the following:
        Focusable content should be disabled or be removed from the DOM

      Issue target: d2l-insights-assessment-quality-dashboard,d2l-table-wrapper,d2l-scroll-wrapper,.d2l-scroll-wrapper-button-right
      Context: <button class="d2l-scroll-wrapper-button d2l-scroll-wrapper-button-right" aria-hidden="true">    
                                        <d2l-icon icon="tier1:chevron-right"></d2l-icon>
                                </button>
      Fix all of the following:
        Focusable content should be disabled or be removed from the DOM
```

Steps to reproduce:
1) Run `d2l-insights-engagement-dashboard.accessibility` test in [this repo](https://github.com/Brightspace/insights-assessment-quality-dashboard). Comment out `aria-hidden-focus` inside `ignoredRules` block

**Comment by dlockhart** [comment]
This is in the new stuff I assume? I guess I can _not_ hide the scroll buttons to screen readers, and then they can just be annoyed that there are buttons there that do nothing of importance to them. This is an example of where fixing an accessibility "bug" is going to make for a worse experience.
